domain/schema/controller/sql/0020-bakery-storage.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE bakery_config (
```

Comment:
What is the primary key on this table? SQLite does provide an automatic one if you don't put one in, but I believe across the rest of domain we are using some sort of primary key. The key can be made up of multiple rows.

Aside from the primary key, should any of these have UNIQUE constraints?

Reply:
> Aside from the primary key, should any of these have UNIQUE constraints?

Nah, we assert this table is a singleton with the index below. Uniqueness on top of this wouldn't add anything